apis/python/src/tiledbsoma/general_utilities.py

```diff
@@ -38,3 +41,22 @@ def get_storage_engine() -> str:
     [lifecycle: experimental]
     """
     return "tiledb"
+
+
+def show_package_versions() -> None:
+    """
+    Nominal use is for bug reports, so issue filers and issue fixers can be on the same page.
+    """
+    line1 = "tiledbsoma.__version__   " + get_implementation_version()
```

Comment:
why not print these immediately? no need to make vars, build a list, join the list, then print the result.

Reply:
@thetorpedodog d'oh